tests/test_parsers.py

```diff
@@ -88,11 +88,9 @@ def test_parse_sssom_dataframe_from_file(self):
 
     def test_parse_sssom_dataframe_from_stringio(self):
         """Test parsing a TSV."""
-        input_path = f"{test_data_dir}/basic.tsv"
-        with open(input_path, "r") as file:
-            input_string = file.read()
-        stream = io.StringIO(input_string)
-        msdf = parse_sssom_table(stream)
+        input_path = test_data_dir.joinpath("basic.tsv")
+        with input_path.open() as file:
+            msdf = parse_sssom_table(file)
```

Comment:
I understand the intention of simplifying the code here (no need to read the entire contents of `basic.tsv` into a string, turn the string into a file-like `StringIO` object, and then pass that object to `parse_sssom_table`, when that function can take a pathname directly), but in this instance I believe this is ill-advised.

Judging from the name of this test (`test_parse_sssom_dataframe_from_stringio`), it is specifically intended to check that you can use `parse_sssom_table` on a `StringIO` object, rather than a file descriptor or a pathname. Now it is no longer testing that.

Reply:
That's a good point, so I updated this in aabdb93.